src/App.css

```diff
@@ -26,7 +27,70 @@ button{
   padding: 20px;
   border-radius: 8px;
   box-shadow: 0 4px 8px rgba(0, 0, 0, 0.1);
-  width: 300px;
+  width: auto;
   margin:0 auto 30px auto;
   align-self: center;
-}
\ No newline at end of file
+}
+
+form {
+  background-color: #ffff;
+  padding: 20px;
+  border-radius: 8px;
+  box-shadow: 0 4px 8px 4px rgba(0, 0, 0, 0.1);
+  width: 300px;
+  margin:0 auto 10px auto;
+  align-self: center;
+  margin-top: 10px;
+}
+
+.TodoInput{
```

Comment:
The CSS classname must be Hyphen Delimited Strings.

Refer: https://www.freecodecamp.org/news/css-naming-conventions-that-will-save-you-hours-of-debugging-35cea737d849/